cmake/multi_image.cmake

```
@@ -82,7 +82,7 @@ function(zephyr_add_external_image_from_source name sourcedir)
     -DBOARD=${IMAGE_BOARD}
     ${IMAGE_KCONFIG_TARGET}
     ${IMAGE_COMMAND_TARGET}
-    -DGENERATE_PARTITION_MANAGER_ENTRY=True
+    -DIS_CHILD_IMAGE=True
```

Same here, let's not invent our own terminology, but use CMake, and call it `IS_EXTERNAL_PROJECT`
```suggestion
    -DIS_EXTERNAL_PROJECT=True
```

---

See https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1508#discussion_r348992530